SequenceBrowser/qSlicerSequenceBrowserModule.cxx

```diff
@@ -253,20 +259,18 @@ void qSlicerSequenceBrowserModule::onNodeRemovedEvent(vtkObject*, vtkObject* nod
   vtkMRMLSequenceBrowserNode* sequenceBrowserNode = vtkMRMLSequenceBrowserNode::SafeDownCast(node);
   if (sequenceBrowserNode)
     {
-    // If the timer is active
-    if (d->UpdateAllVirtualOutputNodesTimer.isActive())
+    // Check if there is any other sequence browser node left in the Scene
+    vtkMRMLScene * scene = qSlicerCoreApplication::application()->mrmlScene();
+    if (scene)
       {
-      // Check if there is any other sequence browser node left in the Scene
-      vtkMRMLScene * scene = qSlicerCoreApplication::application()->mrmlScene();
-      if (scene)
+      vtkMRMLNode * node;
```

Comment:
shouldn't this be `node=` instead?

Reply:
You mean the space between Node and *?  I was just following the example from the scene declaration above.  I guess I can change both.

Reply:
I changed all the "type * variable" declarations in the file to be "type* variable" for consistency.

Reply:
Oh, I see - oops!  Yes, I fixed that.